docker-compose.prod.yml

```diff
@@ -0,0 +1,36 @@
+version: '3.7'
+
+services:
+  rdf-differ:
+    container_name: rdf_differ
+    build:
+      context: .
+      dockerfile: Dockerfile.prod
+    command: gunicorn --bind 0.0.0.0:${RDF_DIFFER_API_PORT} ${FLASK_APP}:app
+    volumes:
+      - ./:/usr/src/app
```

Comment:
we should probably bind a folder path specified in an ENV variable here. 
It should not necessarily be the context folder. 
Moreover, I would recommend binding externally defined volumes (as explained here https://docs.docker.com/compose/compose-file/#volumes).